source/extensions/filters/common/ext_authz/ext_authz_http_impl.cc

```diff
@@ -202,9 +202,20 @@ void RawHttpClientImpl::check(RequestCallbacks& callbacks,
         std::make_unique<Buffer::OwnedImpl>(request.attributes().request().http().body());
   }
 
-  request_ = cm_.httpAsyncClientForCluster(config_->cluster())
-                 .send(std::move(message), *this,
-                       Http::AsyncClient::RequestOptions().setTimeout(config_->timeout()));
+  const std::string& cluster = config_->cluster();
+
+  // It's possible that the cluster specified in the filter configuration no longer exists due to a
+  // CDS removal.
+  if (!cm_.get(cluster)) {
+    // TODO(dio): Add stats related to this.
+    ENVOY_LOG(debug, "ext_authz cluster '{}' does not exist", cluster);
```

Comment:
We will probably need to add tracing here too. I can update https://github.com/envoyproxy/envoy/pull/8142 after you merge it.

Reply:
👍